4.3 - Adding SL Micro 6.0 as supported client (#3057)
uyuni-project/uyuni-docs: modules/client-configuration/pages/supported-features-sl-micro.adoc

[[supported-features-sl-micro]]
= Supported {sl-micro} Client Features

[IMPORTANT]
====
The operating system you run on a client is supported by the organization that supplies the operating system.
{sl-micro} is supported by {suse}.
====

[IMPORTANT]
====

> {sl-micro} is only supported as regular minion ([systemitem]``default`` contact method) for the time being. We are working on managing it as Salt SSH client ([systemitem]``salt-ssh`` contact method), too.

Reviewer: We should avoid future statements in our technical guide.  As a compromise, I suggest:

```suggestion
{sl-micro} is currently supported as a {salt} client with the [systemitem]``default`` contact method only.
```

Author: @keichwa great catch, THANKS.

| {salt}

| CVE auditing (requires CVE ID)
| {salt}

| SCAP auditing
| {question}

| Package verification
| {question}

| Package locking
| {salt}

| System locking
| {question}

| Maintenance Windows
| {question}

| System snapshot
| {cross}

| Configuration file management
| {salt}

| Snapshots and profiles
| {salt}: Profiles supported, Sync not supported

| Power management
| {salt}

| Monitoring server
| {cross}

| Monitored clients {star}{star}
| {salt}

| Docker buildhost
| {cross}

| Build Docker image with OS
| {cross}

| Kiwi buildhost
| {cross}

| Build Kiwi image with OS
| {salt}

| Recurring Actions
| {salt}

| AppStreams
| N/A

| Yomi
| {question}
|===

{star} Virtual Guest Management:

In this table, virtual guest management is split into basic and advanced.

Basic virtual guest management includes listing VMs, slow refresh, VM lifecycle actions (start, stop, resume, pause), and modifying VM vCPU and Memory.

Advanced virtual guest management includes all features of Basic virtual guest management plus fast refresh, VM lifecycle actions (delete, reset, power off), modifying VM disk, network, graphical display, and graphical display configuration.

> {star}{star} On {micro}, only the Node exporter and the Blackbox exporter are available.

Reviewer: ```suggestion
{star}{star} On {sl-micro}, only the Node exporter and the Blackbox exporter are available.
```
